internal/cmd/envoy/shutdown_manager.go

```diff
@@ -170,8 +170,9 @@ func Shutdown(drainTimeout time.Duration, minDrainDuration time.Duration, exitAt
 
 // postEnvoyAdminAPI sends a POST request to the Envoy admin API
 func postEnvoyAdminAPI(path string) error {
+	// TODO: change bootstrap.AdminAddress() to localhost because there're in the same pod?
 	if resp, err := http.Post(fmt.Sprintf("http://%s:%d/%s",
-		bootstrap.EnvoyAdminAddress, bootstrap.EnvoyAdminPort, path), "application/json", nil); err != nil {
+		bootstrap.AdminAddress(egv1a1.IPv4), bootstrap.EnvoyAdminPort, path), "application/json", nil); err != nil {
```

Comment:
this may not right, what about change it to `localhost`? cc @arkodg

Reply:
+1 to `localhost`, good idea

Reply:
+1 `localhost` make things easier